applications/feedback_sentiment_analyzer/DEVELOPMENT.md

```diff
@@ -0,0 +1,76 @@
+# FSA development
+
+This document describes the process for adding a new set of AWS Lambda functions for use with the Feedback Sentiment Analyzer (FSA). The requirements for each individual Lambda function are in [the specification](./SPECIFICATION.md).
```

Comment:
I would not say they are a "new set of AWS Lambda functions". Rather, they are the same functions implemented in a new language.

Reply:
I take your first point, but they don't have to be in a new language. A user could make it in an existing language if they wanted to. I'll noodle on the wording.

For point 2, yes, I'll address that.